textproc/pandoc/Portfile

```diff
@@ -24,33 +25,43 @@ long_description    \
     Textile, groff man pages, plain text, Emacs Org-Mode, AsciiDoc, EPUB (v2 \
     and v3), FictionBook2, and S5, Slidy and Slideous HTML slide shows.
 
-depends_lib-append  \
-                    port:hs-aeson \
-                    port:hs-alex \
-                    port:hs-attoparsec \
-                    port:hs-base64-bytestring \
-                    port:hs-blaze-html \
-                    port:hs-blaze-markup \
-                    port:hs-data-default \
-                    port:hs-extensible-exceptions \
-                    port:hs-happy \
-                    port:hs-highlighting-kate \
-                    port:hs-hslua \
-                    port:hs-http \
-                    port:hs-mtl \
-                    port:hs-network \
-                    port:hs-pandoc-types \
-                    port:hs-parsec \
-                    port:hs-random \
-                    port:hs-syb \
-                    port:hs-sha \
-                    port:hs-tagsoup \
-                    port:hs-temporary \
-                    port:hs-texmath \
-                    port:hs-text \
-                    port:hs-unordered-containers \
-                    port:hs-vector \
-                    port:hs-xml \
-                    port:hs-yaml \
-                    port:hs-zip-archive \
-                    port:hs-zlib
+depends_build-append  \
+                    port:stack
+
+use_configure       no
+
+# the PATH environment must provide /usr/bin/gcc, not ${prefix}/bin/gcc
+build.env \
+    PATH=/usr/bin:$env(PATH) \
+    PREFIX=${prefix} \
+    CC=${configure.cc}
+
+build {
+    xinstall -m 0755 -d ${workpath}/.stack
```

Comment:
You could create this folder in `post-extract`. It will then be easier to just run the three commands nearly out-of-the-box inside configure-build-destroot phase.

You would just override `build.cmd ${prefix}/bin/stack`, `build.target build` and provide whatever `build.args` might be needed.